openmc/deplete/abc.py

```diff
@@ -719,6 +719,9 @@ def __init__(self, operator, timesteps, power=None, power_density=None,
             elif unit.lower() == 'mwd/kg':
                 watt_days_per_kg = 1e6*timestep
                 kilograms = 1e-3*operator.heavy_metal
+                if rate == 0.0:
```

Comment:
The main purpose is to avoid a divide-by-zero below, so I would hesitate to make this check catch small, non-zero values because technically they are still valid.